web/text_layer_builder.css

```diff
@@ -21,7 +21,7 @@
   right: 0;
   bottom: 0;
   overflow: hidden;
-  opacity: 0.2;
+  opacity: 0.3;
```

Comment:
Also, how about using `0.25` as a compromise between the old/new value such that this works OK with both color-values (given the pre-processor comments elsewhere)?